lib/kiev/shoryuken/request_id.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "securerandom"
+require_relative "message_attribute_access"
+
+module Kiev
+  module Shoryuken
+    class RequestId
+      include Kiev::Shoryuken::MessageAttributeAccess
+
+      REQUEST_ID = "request_id"
+      REQUEST_DEPTH = "request_depth"
+      TREE_PATH = "tree_path"
+
+      def call(_worker, _queue, message, _body)
+        request_store = Kiev::RequestStore.store
+        request_store[:request_id] = request_id(message)
```

Comment:
I would suggest to write a decorator class for a message and move request_id, request_depth and friends there. I believe it can even affect `MessageAttributeAccess` module and instead of bunch of utility methods that passes message all the way around you will have 1 simple decorator that incapsulates all of this logic there.